### ISSUE TITLE
Weird FPS issue

### ISSUE THREAD

**Issue by Sekiberius** [tsunamayo/Starship-EVO#3229]
### Short Description of the issue:
was making a giant ship, and finished it, fps was 60+. well, my elevator stopped working so I redid it and now on every single world I go on to I have around 40fps.
...

### Steps to reproduce the issue: I honestly have no clue.

...

### Environment 
- Starship EVO version: 20w46b
- Graphics Card: RTX 3080

### Additional information: I think it may be a memory issue with blueprints as it got worse when I saved my ship. Now the game is using 23% or more of my CPU (R9 3900X) which it wasn't doing before. 

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\` 
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5545756/output_log.txt)

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\

`

**Comment by tsunamayo** [comment]
Hi, what do you mean restart the game? Do you quit to title or quit to windows? Does it still happens if you quit to windows? Thanks

**Comment by Sekiberius** [comment]
Yes, it would still happen no matter what I did, the only solution I could find to fix it was to delete my entire Save_Data folder.

**Comment by tsunamayo** [comment]
@Sekiberius Do you still have the character save so I can have a look at it? It might be coming from other stuff. Did you shoot at your ship? Thanks

**Comment by Sekiberius** [comment]
[Save_Data.zip](https://github.com/tsunamayo/Starship-EVO/files/5610233/Save_Data.zip)
No I didn't shoot it, I destroyed a piston and that's when it all started.

**Comment by tsunamayo** [comment]
@Sekiberius Sorry for the lack of update, I dont know if you are still playing a bit, if yes could you tell me if you had any issue on the Neovox update? Thanks

**Comment by Dreamseizer36b** [comment]
Yes FPS is fine now. I had an issue with 27w37a with severe stuttering, but I loaded back up the save I was working on and it runs fine now as of 21w48b.

**Comment by Dreamseizer36b** [comment]
Yes FPS is fine now. I had an issue with 27w37a with severe stuttering, but I loaded back up the save I was working on and it runs fine now as of 21w48b.
(Had to make a new account as I lost my old password and my old email)
